ui/README.md

```diff
@@ -1,2 +1,37 @@
-# Anbox Cloud - Cloud Gaming Demo
+# Web UI for cloud gaming demo
 
+The Web UI for clouding gaming demo is built with [flutter](https://flutter.dev/).
+
+## Requirements
+
+To setup Web development and build the UI, make sure that you have flutter installed.
+Please follow the [instructions](https://docs.flutter.dev/get-started/install/linux) to install the flutter on your machine.
+
+## Build
+
+To build the UI, simply run:
+
+    ```
+    flutter build web
+    ```
+
+This generates assets, js, html files that are required to run the Web UI and places all of them into the `build/web` folder.
+Please refer to the official [documentation](https://docs.flutter.dev/get-started/web) for more details on how to build Web apps with flutter.
+
+## Deploy
+
+Create a `static` folder under the backend service and copy everything that is in the `build/web` folder to the `static` folder.
+
+    ```
+    mkdir -p backend/service/static
+    cp av build/web/* backend/service/static
```

Comment:
Shouldn't this be:
```suggestion
    cp -av build/web/* backend/service/static/
```

Reply:
The `build` folder would be always under the `ui` folder where people can launch `flutter build web` command to build UI. 
The **full** folder path depends on where people clone and check out this project. It's more like a path relative to the root folder of our project.

Reply:
Yes, sorry, that's what I meant - a path from the root folder.
But then this command won't work, right? It would need to be either `cp -av build/web/* ../backend/service/static/` or `cp -av ui/build/web/* backend/service/static/'.

Reply:
You're right. it would be the latter one after we move this file to the root folder of this project as discussed above.